common/models/workspace.js

```diff
@@ -90,6 +90,7 @@ module.exports = function(Workspace) {
           return {
             name: name,
             description: data.description,
+            supportedLBVersion: data.supportedLBVersion,
```

Comment:
CI fails due to the deep equal assert of template metadata. Will update those test cases too.